lib/tapioca/compilers/dsl_compiler.rb

```diff
@@ -21,12 +21,13 @@ class DslCompiler
         params(
           requested_constants: T::Array[Module],
           requested_generators: T::Array[String],
+          excluded_generators: T::Array[String],
           error_handler: T.nilable(T.proc.params(error: String).void)
         ).void
       end
-      def initialize(requested_constants:, requested_generators: [], error_handler: nil)
+      def initialize(requested_constants:, requested_generators: [], excluded_generators: [], error_handler: nil)
         @generators = T.let(
-          gather_generators(requested_generators),
+          gather_generators(requested_generators, excluded_generators),
```

Comment:
Could we produce an error if the requested/excluded generator names passed do not match any existing generator?

Reply:
Added, see last commit (`Error if requested/excluded generator does not exist`).